unit_test/test_spdm_requester/get_measurements.c

```diff
@@ -20,6 +20,8 @@ static uint8_t m_libspdm_msg_log_buffer[LIBSPDM_MAX_MESSAGE_L1L2_BUFFER_SIZE * 2
 static size_t m_libspdm_opaque_data_size;
 static uint8_t m_libspdm_opaque_data[SPDM_MAX_OPAQUE_DATA_SIZE];
 
+static size_t m_requester_context[SPDM_REQ_CONTEXT_SIZE];
+
```

Comment:
`static size_t m_requester_context[SPDM_REQ_CONTEXT_SIZE];`
type is `size_t` here, why?